lib/cocoapods-core/specification/dsl.rb

```diff
@@ -1486,6 +1486,28 @@ def test_spec(name = 'Tests', &block)
         subspec
       end
 
+      # Represents an app specification for the library. Here you can place all
+      # your app source files for your podspec along with the app dependencies.
+      #
+      # ---
+      #
+      # @example
+      #
+      #   Pod::Spec.new do |spec|
+      #     spec.name = 'NSAttributedString+CCLFormat'
+      #
+      #     spec.app_spec do |app_spec|
+      #       app_spec.source_files = 'NSAttributedString+CCLFormat.m'
+      #       app_spec.dependency 'AFNetworking'
+      #     end
```

Comment:
Ah, ok I see what this is doing. I still think a bit more info would be helpful here but I have a better idea of what's going on based on this

Reply:
Agreed, this documentation will be reflected on the website. I would expand a bit more on the functionality and value app specs provide.

For example, app specs can be used as sample consumers of the library as examples people can follow on how to actually integrate the library.